server/src/main/java/com/netcracker/domain/Message.java

```diff
@@ -0,0 +1,26 @@
+package com.netcracker.domain;
+
+import java.io.Serializable;
+
+public class Message implements Serializable{
```

Comment:
Yes, I use it in my other class, that I don't commit yet, otherwise I'll get NotSerialzableException